nets/Z2Bala.py

```diff
@@ -0,0 +1,84 @@
+"""SqueezeNet 1.1 modified for regression."""
+import torch
```

Comment:
This entire file is a duplicate of:
https://github.com/bddmodelcar/training/blob/master/nets/Z2ColorBatchNorm.py
except w/o the BatchNorm is it not? unless there is some important reason for removing this BatchNorm why did you create this file?